env-hooks/99.ed_object_models.bash

```diff
@@ -16,6 +16,8 @@ function ed_object_models_setup {
 
     model_paths_string="$model_path${model_paths_string:+:${model_paths_string}}"
 
+    export ED_MODEL_PATH=$model_path${ED_MODEL_PATH:+:${ED_MODEL_PATH}}
+
     export GAZEBO_MODEL_PATH=$model_paths_string${GAZEBO_MODEL_PATH:+:${GAZEBO_MODEL_PATH}}
     export GAZEBO_RESOURCE_PATH=$model_paths_string${GAZEBO_RESOURCE_PATH:+:${GAZEBO_RESOURCE_PATH}}
```

Comment:
Just as a tip, all gazebo paths can also be set in the `export` tag of your `package.xml`: http://gazebosim.org/tutorials?tut=ros_plugins